notebooks/concepts/Mosviz concept notebook.ipynb

```diff
@@ -0,0 +1,542 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Additional notebook Glue selection interactions "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If the user wants the \"full power\" of Glue working with the table, a quick pathway is to have them create additional viewers to create subsets.  For example, suppose the `table` has two rows, one for 'rmag' and another for 'gmag', indicating the 'r' and 'g'-band magnitudes of the objects the spectra are of.  The user could then do:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tab = mosapp.get_data('table')\n",
+    "tab['color'] = tab['gmag'] - tab['rmag']  #<- this should update the data table...\n",
+    "\n",
+    "mosapp.scatter2d('color', 'rmag')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now the user can use glue's selection capabilities to draw a box around a part of the color-magnitude diagram, which highlights only those in the MOSViz UI."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Initially this would *replace* the \"small panel\" of the Qt MOSviz with all the glue capabilities.  But eventually we would want this workflow to be possible using the more \"general\" Glue UI when that's working."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Example of a \"mosviz app-specific\" method interactions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In addition to data-loading, one could imagine wanting to implement things like this:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mosapp.rescale_both_spectra()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mosapp.switch_to_spectrum_slits()\n",
+    "#vs\n",
+    "mosapp.switch_to_table_slits()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "spec1d, spec2d, image2d, tablerow = mosapp.get_current_spectrum()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The third cell above in particular is nice because it would otherwise require each viewer being accessed individually, which in the glue-way might be rather awkward to deal with."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Or going even further to specific science workflows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mosapp.compute_redshift_on_all_spectra(...)"
```

Comment:
This starts to become a little too specialized - I'd be more in favor of showing users how to access the ``SpectrumCollection`` and using ``specutils`` (or other packages) to do any analysis.

Reply:
I agree with you but thought I should include it for completeness.